packages/frontend-2/components/header/WorkspaceSwitcher/header/Header.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <div class="p-2 pb-3 flex flex-col gap-y-4">
-    <div class="flex gap-x-2 items-center">
+    <div class="flex gap-x-3 items-center">
       <MenuItem>
-        <NuxtLink :to="to">
+        <NuxtLink :to="to || ''">
```

Comment:
why? isnt it better to not have a `href` at all than a `href` that points to an invalid URL